addons/storyshots/stories/directly_required/index.js

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import { storiesOf } from '@storybook/react'; // eslint-disable-line
 import { action } from '@storybook/addon-actions'; // eslint-disable-line
 
-import Button from '@storybook/components/dist/demo/Button';
+import Button from '@storybook/react/dist/demo/Button';
```

Comment:
it fails at [examples/test-cra/src/storyshots.test.js](https://travis-ci.org/storybooks/storybook/builds/245331027#L696) but after this change. 
[#2833 passed](https://travis-ci.org/storybooks/storybook/builds/244077064)
[#2836 failed](https://travis-ci.org/storybooks/storybook/builds/244080125)

but possible it's more CI issue then this PR?